bluepyopt/electrical/parameters.py

```diff
@@ -93,6 +93,10 @@ def check_bounds(self):
                     (self.name, self._value, str(self.lower_bound),
                      str(self.upper_bound)))
 
+    def destroy(self):
```

Comment:
I think we should wait with all these details until the API is stable.
Like I said I still have to move around Parameters.py (up a level), and other more important API changes are in the pipeline. (All these subclasses of Parameters are probably going to disappear and replaced by NrnParameter, and a new class called Variables is going to be introduced).